test/NuGet.Core.Tests/Microsoft.Build.NuGetSdkResolver.Tests/Microsoft.Build.NuGetSdkResolver.Test.csproj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(NuGetCoreSrcDirectory)Microsoft.Build.NuGetSdkResolver\Microsoft.Build.NuGetSdkResolver.csproj" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="15.6.85" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkPackageVersion)" />
```

Comment:
1) you are missing the import in this file of the dotnetpackageversionpropspath
2) You need to ensure that this property has a default value when "source build" properties are not being injected into this build. likely change it to match the same version as the rest of NuGet.

(Need review and signoff from msbuild folks on this part of the change)

Reply:
Same case as above: This project imports the `commons.test.props` file:

https://github.com/NuGet/NuGet.Client/blob/18580becd553aaa61b7dab4d2b7a3779c4c0b0a2/test/NuGet.Core.Tests/Microsoft.Build.NuGetSdkResolver.Tests/Microsoft.Build.NuGetSdkResolver.Test.csproj#L2

which imports the `config.props` file:

https://github.com/NuGet/NuGet.Client/blob/18580becd553aaa61b7dab4d2b7a3779c4c0b0a2/build/common.test.props#L9

that has the `DotnetPackageVersionPropsPath` project import: 

https://github.com/NuGet/NuGet.Client/blob/18580becd553aaa61b7dab4d2b7a3779c4c0b0a2/build/config.props#L35


The default value is provided in the `config.props` file, modified in this PR.